.github/workflows/docker.yaml

```diff
@@ -2,10 +2,9 @@ name: docker
 
 on:
   workflow_dispatch:
-  pull_request:
   push:
     branches:
-      - main
+      - test/gha-docker
```

Comment:
```suggestion
      - main
```

Reply:
See my comment to skip the last commit when pushing ;-)